mathquill4quill.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-env browser, commonjs */
 
-window.mathquill4quill = function(dependencies) {
+window.mathquill4quill = function (dependencies) {
```

Comment:
Nit: let's remove this diff to keep the history clean and focused to the minimal required changes for this feature.
```suggestion
window.mathquill4quill = function(dependencies) {
```